libs/frontend/feature-build/src/components/toolbar/ToolbarColorPicker.vue

```diff
@@ -8,18 +8,28 @@
       :color="color"
       :gradient="gradient"
       :selectedThemeColors="selectedThemeColors"
+      :resolveThemeVar="resolveThemeVar"
       class="font-color"
-      @selectColor="emit('selectColor', $event.rgba)"
+      @selectColor="emit('selectColor', $event)"
       @applyGradient="emit('applyGradient', $event)"
       @click.stop
     />
   </div>
 </template>
 
 <script lang="ts" setup>
-import { ColorPicker, IRgba } from '@pubstudio/frontend/feature-color-picker'
+import { ColorPicker, IPickerColor } from '@pubstudio/frontend/feature-color-picker'
 import { IThemeVariable } from '@pubstudio/shared/type-site'
-import ToolbarItem from './ToolbarItem.vue'
+import { IThemedGradient } from '@pubstudio/frontend/util-gradient'
+import { resolveThemeVariables } from '@pubstudio/frontend/util-builtin'
+import { ToolbarItem } from '@pubstudio/frontend/ui-widgets'
+import { useBuild } from '../../lib/use-build'
+
+const { site } = useBuild()
+
+const resolveThemeVar = (themeVar: string): string | undefined => {
```

Comment:
To avoid `ColorPicker` dependency on `use-build`, I pass in a function to resolve theme variables.